.env.example

```diff
@@ -14,8 +14,7 @@ KODI_PORT=
 KODI_USERNAME=
 KODI_PASSWORD=
 
-SKILL_APPID=
-SKILL_VERIFY_CERT=
```

Comment:
Flask-Ask does deal with the cert verification -- in fact, I stole the code from there for the stuff I had in here initially ;)

I am not sure if it disables it for Lambda deployments automatically, but that could be one reason for leaving SKILL_VERIFY_CERT in place, so Lambda users can disable it (since it's redundant).

You'd have to check your own deployment though to see if it's verifying the certs.  I just put some print statements in the verifier code to see that it's executing.